build.sbt

```diff
@@ -86,7 +86,7 @@ lazy val commonSettings: Seq[Setting[_]] = Seq(
       s"git@github.com:${bintrayOrganization.value.get}/${name.value}.git"
     )
   ),
-  crossScalaVersions in ThisBuild := Seq("2.11.12", "2.12.10", "2.13.1"),
+  crossScalaVersions in ThisBuild := Seq("2.11.12", "2.12.11", "2.13.2"),
   scalaVersion in ThisBuild := "2.12.9",
```

Comment:
this should also change to 2.12.11, or perhaps to 2.13.2.

Reply:
Good catch, fixed.